packages/integration-sdk-cli/src/visualization/generateVisualization.ts

```diff
@@ -27,9 +27,11 @@ export async function generateVisualization(
     log.warn(`Unable to find any files under path: ${resolvedIntegrationPath}`);
   }
 
-  const { entities, relationships, mappedRelationships } = await retrieveIntegrationData(
```

Comment:
Nothing actually changed in this file. Prettier picked it up. After all of my changes get merged, I'll run prettier on the entire project to prevent this from happening.